primitives/block/src/block.rs

```diff
@@ -231,7 +231,15 @@ impl Block {
             Block::Micro(ref block) => block
                 .body
                 .as_ref()
-                .map(|ex| ex.transactions.iter().map(|tx| tx.fee).sum())
+                .map(|ex| {
+                    ex.transactions
+                        .iter()
+                        .map(|tx| match tx {
+                            ExecutedTransaction::Ok(tx) => tx.fee,
+                            ExecutedTransaction::Err(tx) => tx.fee,
+                        })
```

Comment:
I'd say either
```suggestion
                        .map(|tx| tx.get_raw_transaction().fee()
```
or add `ExecutedTransaction::fee(&self)`